build/scripts/check-nuspec.ps1

```diff
@@ -0,0 +1,83 @@
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try
+{
+    write-host "Verifying contents of Micrsoft.Net.Compilers.nuspec"
+    $deploymentPath = join-path $binariesPath "Deployment\Toolset"
+    [xml]$x = gc (join-path $sourcePath "src\NuGet\Microsoft.Net.Compilers.nuspec")
+    $nuspecFiles = 
+        $x.package.files.file | 
+        %{ $_.src } | 
+        ? { (-not $_.Contains("$")) -and (-not $_.EndsWith(".config")) -and (-not $_.EndsWith("rsp")) } | 
+        %{ split-path -leaf $_ };
+
+    # Files whech are deployed but do not need to be included in the NuGet package
+    $excludedFiles = @(
+        "Toolset_DoNetUse.exe",
+        "Microsoft.Build.dll",
+        "Microsoft.Build.Framework.dll",
+        "Microsoft.Build.Tasks.Core.dll",
+        "Microsoft.Build.Utilities.Core.dll",
+        "Microsoft.DiaSymReader.Native.arm.dll"
+    )
+
+    $deployedFiles = 
+        gci -re -in *.exe,*.dll $deploymentPath |
+        %{ split-path -leaf $_ } |
+        ?{ -not $excludedFiles.Contains($_) };
+
+    if ($deployedFiles.Count -lt 20)
```

Comment:
No good number I could think of.  The exact count seemsn fragile and will require frequent updating.